app/templates/base.html

```diff
@@ -35,7 +35,17 @@
           </ul>
 
           <ul class="nav navbar-nav navbar-right">
-            <li><div id="nav-bar-login"></div></li>
+            {% if not user %}
+              <li>
+                <a href="/api/auth/login">Sign In With <span class="fa fa-github fa-lg"></span></a>
+              </li>
+            {% else %}
+              <li>
+                <button class="btn btn-small btn-outline-success p-0">
```

Comment:
Why is this a button but the sign in is <a>. I recommend <a> by default -- it's a link here!